administrator/language/de-DE/plg_workflow_featuring.ini

```
@@ -4,9 +4,9 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_WORKFLOW_FEATURING="Workflow - Featuring"
-PLG_WORKFLOW_FEATURING_CHANGE_STATE_NOT_ALLOWED="You're not allowed to change the featured state of this item. Please use a workflow transition."
-PLG_WORKFLOW_FEATURING_FEATURED="Featured: %s"
-PLG_WORKFLOW_FEATURING_TRANSITION_ACTIONS_FEATURING_DESC="Define the featured state an item should be set, when executing this transition"
-PLG_WORKFLOW_FEATURING_TRANSITION_ACTIONS_FEATURING_LABEL="Featuring state"
-PLG_WORKFLOW_FEATURING_XML_DESCRIPTION="Add featuring actions to the workflow transitions for your items"
+PLG_WORKFLOW_FEATURING="Workflow – Haupt-Eigenschaft"
+PLG_WORKFLOW_FEATURING_CHANGE_STATE_NOT_ALLOWED="Es ist nicht zulässig, den Haupt-Status dieses Eintrags zu ändern. Bitte einen Workflow-Übergang verwenden."
```

```suggestion
PLG_WORKFLOW_FEATURING_CHANGE_STATE_NOT_ALLOWED="Es ist nicht erlaubt, den Haupt-Status dieses Eintrags zu ändern. Bitte einen Workflow-Übergang verwenden."
```

Einheitlich mit PLG_WORKFLOW_PUBLISHING_CHANGE_STATE_NOT_ALLOWED im publishing_workflow